### PR TITLE
Release candidate v10.5

### PR DESCRIPTION
**Pull request by zivshits**

### CODE REVIEW

File: aws/DRVault-Single-Deployment.json [hunk 1]
```diff
@@ -665,17 +665,15 @@
                             }
                         },
                         "commands": {
-                            "0-enableSSM": {
-                                "command": "powershell.exe -Command \"Set-Service -Name AmazonSSMAgent -StartupType Automatic\"",
-                                "waitAfterCompletion": "0"
```
Comment (nimrody15) [inline aws/DRVault-Single-Deployment.json]: Missing set service to automatic

File: aws/Full-PAS-Deployment.json [hunk 2]
```diff
@@ -2852,7 +2852,7 @@
                 "Assert" : {"Fn::Equals": [{"Ref": "VaultDRPassword"}, {"Ref": "RetypeDRPassword"}]},
                 "AssertDescription": "The password confirmation does not match."
               }
-            ] 
+            ]
```
Comment (nimrody15) [inline aws/Full-PAS-Deployment.json]: Missing support with gov cloud

File: aws/Full-PAS-Deployment.json [hunk 3]
```diff
@@ -2877,7 +2877,7 @@
                     "Parameters": [
                         "VaultInstanceName",
                         "VaultMasterPassword",
-						"RetypeMasterPassword",					   
+						"RetypeMasterPassword",
                         "VaultAdminPassword",
```
Comment (nimrody15) [inline aws/Full-PAS-Deployment.json]: Enable cloudwatch service

File: aws/PAS-Component-Single-Deployment.json [hunk 4]
```diff
@@ -1418,9 +1456,31 @@
...
+						        "AmazonSSMAgent": {
+							        "enabled" : "true",
+                                    "ensureRunning" : "true",
+                                    "files" : ["C:\\Program Files\\Amazon\\SSM\\Plugins\\awsCloudWatch\\AWS.EC2.Windows.CloudWatch.json"]
+							    }
+						    }
+						}
                     }
                 },
                 "AWS::CloudFormation::Designer": {
```
Comment (nimrody15) [inline aws/PAS-Component-Single-Deployment.json]: Can you please remove the designer resource

File: aws/Full-PAS-Deployment.json (hunk 2 above)
Comment (zivshits) [reply aws/Full-PAS-Deployment.json]: Done

File: aws/Vault-Single-Deployment.json [hunk 5]
```diff
@@ -808,43 +800,62 @@
...
+                            "windows": {
+                                "AmazonSSMAgent": {
+                                    "enabled" : "true",
+                                    "ensureRunning" : "true",
+                                    "files" : ["C:\\Program Files\\Amazon\\SSM\\Plugins\\awsCloudWatch\\AWS.EC2.Windows.CloudWatch.json"]
+                                }
+                            }
                         },
                         "commands": {
                             "1-restartSSM": {
```
Comment (nimrody15) [inline aws/Vault-Single-Deployment.json]: Not a blocker, change number to 01, 02

**Pull request merged**